### PR TITLE
custom sound notif for messages and mentions

### PR DESCRIPTION
**Pull request by ftwpala**
…torage

Issues that might come up: 
-- Uploaded file is very big
-- Sound is played twice

### CODE REVIEW

File: assets/chat/js/settings.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ function upgradeSettings(chat, oldversion, newversion) {
             chat.settings.delete("allowNotifications");
         }
 
+        chat.settings.set("soundnotificationhighlight", arr);
+        chat.settings.set("soundnotificationwhisper", arr);
+
+        chat.settings.set("notificationsoundfile", arr);
+
```
Comment (jbpratt78) [inline assets/chat/js/settings.js:42]: can we remove these empty lines

File: webpack.config.js [hunk 2]
```diff
@@ -262,6 +263,13 @@ module.exports = {
                 exclude: path.resolve(__dirname, 'node_modules/font-awesome/'),
                 loader: 'file-loader',
                 options: { name: 'img/[name].[md5:hash:base64:6].[ext]' }
+            },
+            {
+                test: /\.(ogg|mp3|wav|mpe?g)$/i,
```
Comment (jbpratt78) [inline webpack.config.js:268]: Do all of the file types work? If so, lets add a message somewhere noting the valid extensions, if not lets trim this down to just valid.

Comment (jbpratt78) [reply webpack.config.js:268]: @ftwpala

Comment (ftwpala) [reply webpack.config.js:268]: @jbpratt78 oh right, sec

**Pull request merged**